tests/providers/azure/services/iam/iam_custom_role_permits_administering_resource_locks/iam_custom_role_permits_administering_resource_locks_test.py

```diff
@@ -0,0 +1,96 @@
+from unittest import mock
+from uuid import uuid4
+
+from azure.mgmt.authorization.v2022_04_01.models import Permission
+
+from prowler.providers.azure.services.iam.iam_service import Role
+
+AZURE_SUSCRIPTION = str(uuid4())
```

Comment:
To be consistent in the tests we can include that in a fixture file and reutilise it all over the code.

Reply:
Done!